Updated BRMS and BPM references (#348)
redhat-documentation/supplementary-style-guide: supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

*See also*:

// BxMS: Added "In Red Hat Process Automation Manager,"
[discrete]
[[scorecard]]
==== image:images/yes.png[yes] Scorecard (noun)

> *Description*: In Red Hat Process Automation Manager, _Scorecard_ is a risk management tool that is a graphical representation of a formula used to calculate an overall score. It is mostly used by financial institutions or banks to calculate the risk they can take to sell a product in the market. It can predict the likelihood or probability of a certain outcome. Red Hat Process Automation Manager supports additive scorecards that calculates an overall score by adding all partial scores assigned to individual rule conditions.

Reviewer: ```suggestion
*Description*: In Red Hat Process Automation Manager, _Scorecard_ is a risk management tool that graphically represents a formula used to calculate an overall score. It is mostly used by financial institutions or banks to calculate the risk they take when selling a product. Scorecards help predict the likelihood or probability of a certain outcome. Red Hat Process Automation Manager supports additive scorecards that calculates an overall score by adding all partial scores assigned to individual rule conditions.
```
- Consider editing for clarity.
